libs/application-generic/src/usecases/workflow/update-workflow/update-workflow.usecase.ts

```diff
@@ -753,6 +757,16 @@ export class UpdateWorkflow {
           workflowType: command.type,
         }),
       );
+
+      await this.deleteControlValuesUseCase.execute(
```

Comment:
when the step is removed --> remove also the control values

Reply:
Nit-picky one: The deleteControlValues use case is just a single delete call to a repository. The existence of a use case should be justified by the use case doing at least two things. So, we could replace it with a repository call and remove two extra files (use case & command)

Reply:
I'm ok with that.